config/initializers/session_store.rb

```diff
@@ -10,7 +10,7 @@
 
   session_options = {}
   if MiqEnvironment::Command.is_appliance?
-    session_options[:secure]   = true unless MiqEnvironment::Command.is_container?
+    session_options[:secure]   = true unless MiqEnvironment::Command.is_container? || ENV['MIQ_ALLOW_HTTP_COOKIES'] == "true"
```

Comment:
@kruge002 Would you mind changing this to:
```ruby
session_options[:secure]   = true unless ENV["ALLOW_INSECURE_SESSION"]
```
There's one use case for the container check and I'd prefer to move this to an ENV variable instead.  I can make the corresponding PR for that change.